src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -7,13 +7,15 @@
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
-    public static final String MESSAGE_INVALID_RECIPE_DISPLAYED_INDEX = "The recipe index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid \n %s";
+    public static final String MESSAGE_INVALID_RECIPE_DISPLAYED_INDEX = "The recipe index provided is invalid \n %s";
     public static final String MESSAGE_INVALID_STEP_DISPLAYED_INDEX = "The step index provided is invalid";
```

Comment:
These messages are used when the index is invalid right? Do we need to display the whole command usage for an index typo? Also, should we change the invalid step message to include the command usage as well?

Reply:
I'm conflicted about displaying the whole message too, but I went ahead with that since that was the case for some commands. I think we only have to include the whole message when we throw a CommandException? On the other hand, a ParseException could be due to the user typing the command wrongly too.